python/torch_mlir/eager_mode/torch_mlir_dispatch.py

```diff
@@ -123,7 +126,11 @@ def build_script_function(
     else:
         graph.registerOutput(node.output())
 
-    fn_name = str(node).strip()
+    # The first alternation handles non-alpha characters at the beginning.
+    # MLIR doesn't support such symbol names unless quoted, hence safer to remove.
+    # The second alternation handles non-word characters in the rest of the name (eg =, :, etc).
+    # LLVM (and therefore IREE) doesn't support such symbol names.
```

Comment:
My understanding is that LLVM allows arbitrary symbol names too. Are you sure you need this? Also, can you remove the mention of IREE? IREE is not a part of Torch-MLIR.

Reply:
addressed in the PR that this change is associated with (IREE removed there)